node/builder.go

```diff
@@ -224,12 +224,16 @@ var LibP2P = Options(
 )
 
 func IsType(t repo.RepoType) func(s *Settings) bool {
-	return func(s *Settings) bool { return s.nodeType == t }
+	return func(s *Settings) bool { return s.nodeType.Type() == t.Type() }
```

Comment:
Do we actually need the `.Type()` here?

Reply:
Now that we have singleton for `nodeType`, we don't. Removing it.